Error installing headscale admin

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Headscale

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/headscale.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Installation of headscale admin fails with errors



### 🔄 Steps to reproduce the issue.

install headscale with admin option

### ❌ Paste the full error output (if available).

```
Processing triggers for libc-bin (2.36-9+deb12u10) ...
0: line 26: PORT: unbound variable

[ERROR] in line 26: exit code 0: while executing command cat <<EOF > /etc/caddy/Caddyfile
:{$PORT:80}

handle_path /admin* {
        root * /opt/headscale-admin
        encode gzip zstd

        # Correct MIME types for JS/WASM
        header {
                @js_files path *.js
                @wasm_files path *.wasm

                Content-Type @js_files application/javascript
                Content-Type @wasm_files application/wasm

                X-Content-Type-Options nosniff
        }

        # Fallback for SPA routing
        try_files {path} {path}/ index.html
        file_server
}

EOF


[ERROR] in line 1281: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/"$var_install".sh)" $?
```

### 🖼️ Additional context (optional).

_No response_

--- a/install/headscale-install.sh
+++ b/install/headscale-install.sh
@@ -23,7 +23,7 @@ if [[ ${prompt,,} =~ ^(y|yes)$ ]]; then
   $STD apt-get install -y caddy
   $STD caddy stop
   rm /etc/caddy/Caddyfile
-  cat <<EOF >/etc/caddy/Caddyfile
+  cat <<'EOF' >/etc/caddy/Caddyfile
 :{$PORT:80}
 
 handle_path /admin* {

